moochub-schema.json

```
@@ -1,555 +1,3105 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://github.com/moochub/schema/releases/tag/v2.3",
+  "$id": "https://github.com/moochub/schema/releases/tag/v3.0",
   "title": "JSON schema for MOOC providers integrated into moochub.org",
   "description": "This schema specifies the JSON format each MOOC provider must offer in order to be integrated into moochub.org",
   "type": "object",
   "properties": {
-    "links": {
-      "title": "Links",
+    "id": {
+      "type": "string",
+      "description": "The ID of the course as an UUID.",
+      "minLength": 1,
+      "example": "8978f51e-770c-4d0a-a2e3-23fae8f6aa79"
+    },
+    "type": {
+      "type": "string",
+      "description": "Type of the object.",
+      "enum": [
+        "Course"
+      ]
+    },
+    "attributes": {
       "type": "object",
-      "description": "Page information for the API",
+      "description": "The Attributes of the object.",
```

```suggestion
      "description": "The attributes of the course.",
```

This is course-specific, isn't it?

---

true and agreed to change to course